testsuite/tests/kuadrant/authorino/response/test_headers.py

```diff
@@ -16,11 +16,11 @@ def header_name(request):
 def authorization(authorization, header_name):
     """Add response to Authorization"""
     authorization.responses.clear_all()  # delete previous responses due to the parametrization
-    authorization.responses.add_json("header", [Property("anything", Value("one"))], wrapper_key=header_name)
```

Comment:
`operator.http.conftest` also requires a change with wrapper_key to work properly

Reply:
Fixed